src/validations/rules/sar.sch

```diff
@@ -244,7 +244,7 @@
                 see="https://github.com/18F/fedramp-automation/issues/348"
                 test="
                     if (@uuid eq $end-uuid) then
-                        xs:dateTime(descendant::oscal:collected) gt $P180D
+                        xs:dateTime(descendant::oscal:collected[1]) gt $P180D
```

Comment:
This could be totally wrong, but it masks this issue when applying the compiled Schematron to the SAR template:

```
Required cardinality of first argument of xs:dateTime() is zero or one; supplied value contains 10 items (xs:untypedAtomic('2020-10-10T00:00:00Z'), xs:untypedAtomic('2020-10-10T00:00:00Z'),... )
```